4/bin/entrypoint.sh

```diff
@@ -21,5 +21,9 @@ if [ "${SONAR_PROJECT_BASE_DIR:-}" ]; then
 fi
 
 export SONAR_USER_HOME="$PROJECT_BASE_DIR/.sonar"
-sonar-scanner "${args[@]}"
 
+if [ "${1#-}" != "${1}" ] || [ -z "$(command -v "${1}")" ]; then
+  set -- sonar-scanner "${args[@]}"
```

Comment:
Would be nice to have the possibility to pass parameters directly, like -Dsonar.projectVersion=foo, which is not possible right now. Adding $* to the end of command would allow that. It also makes possible to overwrite any setting coming from environment variables since last parameters have precedence.
`  set -- sonar-scanner "${args[@]}" $*`